test/testdata/deployednettemplates/recipes/README.md

```diff
@@ -0,0 +1,45 @@
+# Recipes
+
+Most of the recipes' net.json and genesis.json use one of the following methods to call `netgoal generate`:
+1. `Makefile`
+2. `python3 {GO_ALGORAND_PATH}/test/testdata/deployednettemplates/generate-recipe/generate_network.py -f {PATH_TO}/network-tpl.json`
+
+Details for netgoal generate could be found in the binary with:
+```
+netgoal generate -h
+```
+
+Source code for netgoal can be found in `{GO_ALGORAND_PATH}/cmd/netgoal/generate.go`
+
+Quick reference when reading some of the Makefiles in this directory:
+```
+PARAMS=-w 100 -R 8 -N 20 -n 100 -H 10 -X 10 --node-template node.json --relay-template relay.json --non-participating-node-template nonPartNode.json
+```
+- If you specify `-n` (number of participating algod nodes), the default if you don't pass in `-N` (number of machines to host the algod nodes) is set to match `-n`, i.e. one algod node per machine. To increase the number of nodes per machine, specify both `-n` and `-N`. `-n 10 -N 5` means 10 participating algod nodes across 5 machines, so 2 algod nodes per machine.
+- If you specify `-X` (number of non-participating nodes, npn), the default if you don't pass in `-H` (number of machines to host the npn) is set to the match `-X`, i.e. one npn per machine. To increase the number of nodes per machine, specify both `-X` and `-H`. `-X 10 -H 5` means 10 npns across 5 machines, so 2 npns per machine.
+
+```
```

Comment:
Maybe this belongs in `cmd/netgoal/README.md` ?

Reply:
Sounds good. I'll move it there.

Reply:
Moved details and examples to cmd/netgoal/README.md